runtime/src/main/java/org/corfudb/protocols/wireprotocol/TokenType.java

```diff
@@ -14,7 +14,7 @@
  * Created by dalia on 4/8/17.
  */
 @RequiredArgsConstructor
-public enum TokenType implements ICorfuPayload<TokenType> {
+public enum TokenType implements ICorfuPayload {
```

Comment:
The `implements ICorfuPayload`, along with the serialization/deserialization methods are no longer needed.

Reply:
I got rid of `ICorfuPayload`.